docs/general/contributing.md

```diff
@@ -68,19 +68,111 @@ There are a few basic ground-rules for contributors:
 4. **Significant modifications**, even by contributors, ought to be subject to a **pull request** to
    solicit feedback from other contributors.
 5. Pull requests to solicit feedback are _encouraged_ for any other non-trivial contribution but
-   left to the discretion of the contributor.
-6. Contributors should attempt to adhere to the prevailing `MarkDown` style, language, and layout.
+   left to the contributor’s discretion.
+6. Contributors should adhere to the prevailing `MarkDown` style, language, and layout.
 7. Correct grammar should be used at all times. Pull requests with typos will not be merged until
    fixed.
 8. Care should be taken to remain as objective and informative as possible. There should be no
    editorializing, and external bias should not be present.
-9. We use a plugin called Prettier to standardize the style across documents. You can run this on
-   your local copy with `npx lint-staged`, but for simplicity we also have a bot which runs this for
-   us in your PRs.
+9. We use the Prettier plugin to standardize the style across documents. You can run this on your
+   local copy with `npx lint-staged`, but for simplicity, we also have a bot that runs this for us
+   in your PRs.
+
+## Style Guides
+
+:::info Wiki General Style Guides
+
+The style guide from the
+[Substrate Knowledge Base](https://github.com/substrate-developer-hub/knowledgebase/blob/master/CONTRIBUTING.md#documentation-style)
+provides general guidelines about how to write your Wiki contribution. See also
+[the Wiki readme file](https://github.com/w3f/polkadot-wiki?tab=readme-ov-file#contributing-to-documentation).
+
+:::
+
+The Polkadot Wiki is powered by [Docusaurus](https://docusaurus.io/). Docusaurus 3 was a major
+version upgrade that caused [MDX errors](https://docusaurus.io/docs/3.2.1/migration/v3). As a
+general rule, avoid using curly brackets and special characters. If you must use them, for example,
+to get the \<-\> symbol, you can escape the error with a `\`.
+
+A consequence of the Docusaurus 3 upgrade is the removal of the conditional rendering with Kusama
+and system chains. If you need to display text for different chains, use `Tabs` and `TabItem`
+formatting.
```

Comment:
It is okay to just say we moved away from conditional rendering. 

It was a custom built feature by us and it is possible to have conditional rendering with Docusaurus 3. My understanding was that it was not worth the effort, given the decision to remove Kusama guide. 

The code for conditional rendering is here:
https://github.com/w3f/polkadot-wiki/blob/master/scripts/injectPlugin.js